### ISSUE TITLE
codebuild/common-posix.sh missing

### ISSUE THREAD

**Issue by Bu11etmagnet** [awslabs/aws-c-common#381]
All macOS builds failed in Travis with:
```
$ ./codebuild/common-posix.sh -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
/Users/travis/.travis/functions: line 104: ./codebuild/common-posix.sh: No such file or directory
The command "./codebuild/common-posix.sh -DCMAKE_EXPORT_COMPILE_COMMANDS=ON" exited with 127.

```

**Comment by Bu11etmagnet** [comment]
For example, https://travis-ci.org/awslabs/aws-c-common/jobs/537375512

**Comment by justinboswell** [comment]
@ColdenCullen is working on that, the shell scripts were removed in favor of a unified build runner. When he commits, you should just need to update your branch and Travis will start working again.

**Comment by ColdenCullen** [comment]
Will be closed by #370.
